src/components/PaginationButtons.js

```diff
@@ -0,0 +1,108 @@
+const React = require('react');
+
+const ButtonGroup = require('./ButtonGroup');
+
+const StyleConstants = require('../constants/Style');
+
+const { buttonTypes } = require('../constants/App');
+
+const PaginationButtons = React.createClass({
+  propTypes: {
+    currentPage: React.PropTypes.number.isRequired,
+    onClick: React.PropTypes.func,
+    pageRange: React.PropTypes.number,
+    primaryColor: React.PropTypes.string,
+    styles: React.PropTypes.object,
```

Comment:
Is there a reason this is `styles` instead of `style`? I think this should be renamed to `style` to match the [React naming convention](https://facebook.github.io/react/docs/dom-elements.html#style).